common.mk

```diff
@@ -24,7 +24,7 @@ export JEST_ENV := $(call env_or_else_default,JEST_ENV)
 
 .PHONY: start-server
 start-server:
-	cellxgene launch -p $(CXG_SERVER_PORT) $(CXG_OPTIONS) $(DATASET)
+	PROJECT_ROOT=$(PROJECT_ROOT) python -m server.cli.launch
```

Comment:
Shouldn't this call the `launch_server.sh` script and pass through all the args?
Or is this tricky with make files... it might be. Wondering if we should just remove this make target and use the script if so.

Reply:
ok cool, that makes sense